SPECS/kernel/kernel.signatures.json

```diff
@@ -1,7 +1,7 @@
 {
  "Signatures": {
   "config": "681e8ff274fc3fc79eaf38d5152e99702e22346bd2aaff06925715dcf1a00fc3",
-  "config_aarch64": "e5e5d73cb64189a18c4129d466545f81b8cb8fe8ceeacddeb39b6e73e1822fc1",
+  "config_aarch64": "48bcc7da9cb3da659da1d69ce9c07258553b2da398589464ceaf3e5893efdd7a",
```

Comment:
Reminder to update the hash after finalizing the config changes

Reply:
updated